packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,131 @@ related: ['Button']
+Users may need to provide additional input to complete an action. To accomplish this, use forms or other elements within the modal, like in a [modal wizard](/components/modal/react#with-wizard).
 
-Some usage examples include:
-* Notifying a user that an operation may take a long time to complete
-* Notifying a user that a critical process is complete
+## Content
 
-**Note:** If information is not critical, don’t use a passive dialog. Instead, use inline notifications or toast notifications.
+All modal content should be descriptive and specific so that users can scan, understand an action's context, and make a quick decision.
 
-#### Get required user input
-Sometimes users may need to provide additional input in order to complete an action. In this situation a form or other type of element may be incorporated into the modal. An example of this use case is the [modal wizard](/components/modal/react#with-wizard).
+A modal is comprised of three main content sections:
 
-### Content
-* Clearly restate the action the user is trying to take and explicitly outline the consequences of that action.
+1. **Headline:** A headline introduces a modal’s purpose, usually as a question. Use important key words like “permanently” to contextualize an action and its impact.
 
-* Provide button labels that make it simple for users to understand and confirm their choices. Use descriptive verbs that clearly indicate the consequence of a selection, like Delete, Save, or Create. Avoid vague labels like OK or Done.
+2. **Body text:** Body text provides additional information about an action’s consequence in three lines or less, typically concerning changes to a user’s workflow or access to information. If a modal’s headline fully communicates an action’s context and consequences, body text isn’t needed.
 
-* Use descriptive title and button labels so the user can scan content and make a decision quickly. Ambiguous title and button labels make it more difficult for the user to understand their options and make a decision.
+3. **Buttons:** Buttons allow a user to answer the headline prompt or question. Write button labels as action-based verbs to clearly communicate their function and consequence, and keep them to 1–3 words. Often, verbs for buttons can be pulled directly from a modal’s headline to increase scannability. <br /><br />
+Example headline: *Permanently delete database?*<br />
+Example buttons: *Delete* and *Cancel* <br /><br />
+For more button guidance, see our [button label guidelines](/components/button/design-guidelines#content).
 
-* If the modal needs to convey the importance of information visually, icons can be added.
+For general content guidance to help write effective modal dialogs, see our [UX writing style guide](/ux-writing/about).
 
-See our [content guidelines](/ux-writing/about) for additional guidance.
 
 #### Icon use in modal dialogs
```

Comment:
I feel like we should add a sentence under the header to mention that these are optional and can be used as the use case requires